nano/core_test/unchecked_map.cpp

```diff
@@ -249,9 +249,9 @@ TEST (unchecked, multiple_get)
 		unchecked1.push_back (i.block->hash ());
 	}
 	// Asserts the payloads where correclty saved
-	ASSERT_TRUE (std::find (unchecked1.begin (), unchecked1.end (), block1->hash ()) != unchecked1.end ());
```

Comment:
Leave it this as to was, iterators and equality asserts could cause problems. Leave as it was .